docs/getting-started-on-community-edition/quick-start/index.md

```diff
@@ -87,6 +87,24 @@ The sections below will guide you through setting up your destination to receive
 | BigQuery | — | :white_check_mark: | — |
 | Synapse Analytics | — | — | :white_check_mark: |
 
+## Real-Time Streaming Storage Options
```

Comment:
Should be "Real-time streaming storage options" (note the lower case letters) for consistency with other titles...